### ISSUE TITLE
Variance cutoff used for Noise Injection

### ISSUE THREAD

**Issue by vinbhaskara** [jixiaozhong/RealSR#14]
Can you please answer 5 questions because I could not find the details in your paper:
1) What is the variance cutoff you used for selecting the noise patches
2) What were the sizes of those noise patches while training (i.e. LR resolution while training).
3) Do you apply the variance cutoff on the variance of gray scale image or something else?
4) How many kernels and noise patches did you have in your Pool?
5) What is the Clean Up scale factor used?

Thanks!

**Comment by vinbhaskara** [comment]
@jixiaozhong  Could you please respond? These settings were not made clear in the paper for reproducibility purposes.

**Comment by yoon28** [comment]
Since the paper did not present the parameter settings in detail, I also want to know the secret. In addition to @bsvineethiitg 's five questions, how do you set the initial learning rate and learning rate schedule?

**Comment by voyager0003** [comment]
I guess they need waiting for the patent, just keep patient

**Comment by yoon28** [comment]
@bsvineethiitg I implemented the paper unofficially. I saw that noises are very well removed but the sharpness of my results is not comparable to the original results. As you know, reproducing the paper is difficult because the author did not share many of the hyper-parameter settings. 
https://github.com/yoon28/realsr-noise-injection

**Comment by voyager0003** [comment]
@yoon28 That's true. I implemented the paper using KernelGAN to estimate kernel and create a kernel pool, then using the ESRGAN to train paired images. However, my results is not comparable to the original results too

**Comment by qibao77** [comment]
> @yoon28 That's true. I implemented the paper using KernelGAN to estimate kernel and create a kernel pool, then using the ESRGAN to train paired images. However, my results is not comparable to the original results too

+1

**Comment by yoon28** [comment]
The author unveiled the training code. 
Check this out.
https://github.com/Tencent/Real-SR
After I read the author's implementation, I confirmed that my code was quite close to the author's except for some important hyperparameters. 
I appreciate authors to share their codes.
